### DISCUSSION TITLE
"Invalid custom operator" when i'm trying to use custom operators in my Inspector

### DISCUSSION THREAD

**Discussion by Areave** [clientIO/joint#2507] [Q&A]
### Introduction

Hello! I am trying to show object field only if other object field is filled by string. Didn't found proper operator so decided to make custom one. But it doesnt work. Even if i use example from documentation (from "Custom Operators in Expressions") it still doesnt work, showing error.

![image](https://github.com/clientIO/joint/assets/45808770/d1902423-0216-4a8c-9225-4bb6250bd9e1)

How can i solve my problem about showing field by condition?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Are you passing `operators` to the Inspector constructor somewhere?

**Reply by Areave** [reply]
nope, just in config, as on the screen

**Reply by kumilingus** [reply]
And how do you use your config?
You need to pass both `inputs` and `operators` to the `ui.Inspector` constructor (or the `ui.Inspector.create()` static method).
